src/components/ReactPlayer/ReactPlayer.tsx

```diff
@@ -308,38 +294,38 @@ export const ReactPlayerBlock = React.forwardRef<ReactPlayerBlockHandler, ReactP
             setEnded(true);
         }, [loop, playerRef]);
 
-        const renderCustomBarControls = useCallback(
-            (isMuted: boolean, elapsedTimePercent: number) => {
-                if (controls !== MediaVideoControlsType.Custom || !isPlaying) {
-                    return null;
-                }
+        const onPlayClick = useCallback(() => {
```

Comment:
could you please check this updates in pages of this site https://scale.yandex.ru/2022 (and on mobile devices too)?

Reply:
I couldn't, because all the videos in the website moved to youtube version videos. But seems work correctly on mobile devices